src/Features/VolumetricLighting.cpp

```diff
@@ -66,15 +159,83 @@ void VolumetricLighting::DataLoaded()
 void VolumetricLighting::PostPostLoad()
 {
 	if (REL::Module::IsVR()) {
-		if (settings.EnabledVL)
+		if (settings.ExteriorEnabled || settings.InteriorEnabled)
 			EnableBooleanSettings(hiddenVRSettings, GetName());
 		auto address = REL::RelocationID(100475, 0).address() + 0x45b;  // AE not needed, VR only hook
 		logger::info("[{}] Hooking CopyResource at {:x}", GetName(), address);
 		REL::safe_fill(address, REL::NOP, 7);
 		stl::write_thunk_call<CopyResource>(address);
+
+		// Skip volumetric lighting rendering
+		REL::safe_write(REL::RelocationID(35560, 0).address() + REL::Relocate(0x254, 0), &REL::JMP8, 1);
+		// Move it to render after depth to ensure camera matches rest of scene
+		stl::write_thunk_call<RenderDepth>(REL::RelocationID(35560, 0).address() + REL::Relocate(0x2EE, 0));
 	}
+
+	bEnableVolumetricLighting = reinterpret_cast<bool*>(REL::VariantOffset(0x3232EF0, 0x338C544, 0x3485362).address());
```

Comment:
Offsets are disfavored since they will break if another version comes out. Can you get address ids instead? As you know, you only need 2 since 1.5.97 and VR will match (with a PR to vr address lib).

Reply:
For some reason I didn't think globals would have IDs, updated and PR issued for the VR db